generate-docker-env.sh

```diff
@@ -78,7 +90,7 @@ SUPERUSER_PASSWORD=${SUPERUSER_PASSWORD:-password}
 # *including* the trailing slash
 #
 # leave blank for local images
-IMAGE_REGISTRY=${IMAGE_REGISTRY:-ghcr.io/gatenlp/}
+IMAGE_REGISTRY=${IMAGE_REGISTRY-ghcr.io/gatenlp/}
```

Comment:
This and a number of other settings can legitimately be set to the empty string, so I've switched from `:-` (default if unset _or if set to the empty string_) to just `-` (default if unset, but if set to empty then leave empty)